lib/livebook_web/live/session_live.ex

```diff
@@ -619,7 +619,7 @@ defmodule LivebookWeb.SessionLive do
     name = normalize_name(name)
     Session.set_notebook_name(socket.assigns.session.pid, name)
 
-    {:noreply, socket}
+    {:noreply, assign(socket, page_title: name)}
```

Comment:
This event corresponds to the current user changing the title, but we want to reflect when other users do it as well. All changes are received as operations, so instead I'd add `after_operation` (like [this](https://github.com/livebook-dev/livebook/blob/6d82e9e53d0b6825ceb94f0a25e32ac60ddc0d9f/lib/livebook_web/live/session_live.ex#L1069)) for `{:set_notebook_name, _client_pid, name}` and assign the new title there.

Also, we should reflect the notebook name when navigating to another notebook and revert back to Livebook when moving elsewhere. To do that we can set `page_title: "Livebook"` when mounting other live views, and for this specific one it would be "Livebook - Notebook name".

Let me know if anything is unclear :)

Reply:
I think I got it this time